spec/README.md

```diff
@@ -0,0 +1,100 @@
+### UTM Adapter Specification
+
+This document details at a high level the requirements for a UTM adapter that will be integrated in QGCS codebase, on the UTM Service provider side, the interfaces . The **Deliverables** detail specific outputs that will be developed as a part of this effort. 
+
+## Background
+UTM services or U-space services as it is called in the EU are a set of digital services to enable safe integration of drones with manned aviation that are regulated and will be verified and certified in the EU. Automated verification of the ability to conform to UTM standards forms a part of the UTM Service provider (USSP) certification framework. 
+
+The EU U-space law builds a regulatory framework for UTM, it is expected that other jurisdictions will have their own version of UTM regulations, while this work at this time (June 2023) uses language and services as described / used in the EU U-Space law, it is not limited to it. It is expected that the codebase will change as other jurisdictions will build a regulatory / legal framework for integrating drones in the airspace and enabling "advanced" operations like BVLOS etc. 
+
+### UTM Standards 
+The [EU U-Space law](https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A32021R0664) and the associated [acceptable means of compliance (AMC) and guidance material (GM)](https://www.easa.europa.eu/en/document-library/acceptable-means-of-compliance-and-guidance-materials/amc-and-gm-implementing) has been published. 
+- [Network Remote ID](https://www.astm.org/f3411-22a.html): Mentioned in the AMC/GM document as a acceptable way to comply to Net-RID. This standard specifies interfaces to ensure that 
+- [EuroCAE ED-269](https://eurocae.net/news/posts/2020/june/ed-269-minimum-operational-performance-standard-for-uas-geo-fencing/): Mentioned in the AMC/GM document as a acceptable way to share with Geozone information.
+- [USSP Interoperability](https://www.astm.org/f3548-21.html): Sharing operational intents / flight authorization can be shared via this standard, the `interuss/dss` implements this standard. 
+
+### Services with no / lack of standards
```

Comment:
There is some activity in ISO, but how aviation will ingest in is TBD.

Reply:
No action at this point